client/http/api.go

```diff
@@ -79,6 +79,7 @@ const (
 	Status              = "/pd/api/v1/status"
 	Version             = "/pd/api/v1/version"
 	operators           = "/pd/api/v1/operators"
+	safepoint			= "pd/api/v1/gc/safepoint"
```

Comment:
Please align with other definitions:

```suggestion
	safepoint			= "/pd/api/v1/gc/safepoint"
```